cypress/plugins/index.js

```diff
@@ -1,24 +1,3 @@
-const wp = require('@cypress/webpack-preprocessor');
 module.exports = on => {
-  const options = {
-    webpackOptions: {
-      resolve: {
-        extensions: ['.ts', '.tsx', '.js'],
-      },
-      module: {
-        rules: [
-          {
-            test: /\.tsx?$/,
-            use: [
-              {
-                loader: 'ts-loader',
-                options: {transpileOnly: true},
-              },
-            ],
-          },
-        ],
-      },
-    },
-  };
-  on('file:preprocessor', wp(options));
```

Comment:
do we not need this anymore?

Reply:
nvm, just saw the PR description, nice!

Reply:
Yeah. The file needs to stick around and export something, otherwise Cypress complains, Leaving the function block empty causes ESLint to complain. So there is just a comment now